src/storage/migrations.ts

```diff
@@ -121,7 +121,53 @@ const removeCurrentNetworkAccounts: Migration = {
   },
 };
 
+const flattenBalances: Migration = {
```

Comment:
Do we really need to transfer the balance data, because they will still be fetched by the background task?!

Reply:
Why not?